config/routechangerconfig.go

```diff
@@ -6,17 +6,25 @@ import (
 	"strings"
 )
 
-type routeChangerConfig struct {
+type routeChangerConfigItem struct {
 	Reg  *regexp.Regexp
 	Repl string
 	Sep  string
 }
 
+type routeChangerConfig []routeChangerConfigItem
+
 func (rcc routeChangerConfig) String() string {
-	if rcc.Reg == nil {
-		return ""
+	b := strings.Builder{}
+	for i, rcci := range rcc {
+		if i > 0 {
+			b.WriteString("\n")
+		}
+
+		fmt.Fprintf(&b, "%s%s%s%s%s", rcci.Sep, rcci.Reg,
+			rcci.Sep, rcci.Repl, rcci.Sep)
```

Comment:
I would say implement Stringer interface in routeChangerConfigItem and use it

```
func (rcci routeChangerConfigItem) String() string {
    return rcci.Sep + rcci.Reg + rcci.Sep + rcci.Repl + rcci.Sep
}

// and then use
b.WriteString(rcci.String())
```

Reply:
alternatively you can also use Sprintf to return, but format string is more costly and we seem not to really need it other than using the regexp as string as we can do by calling String().